ksqldb-engine/src/main/java/io/confluent/ksql/util/QueryMetadata.java

```diff
@@ -297,16 +309,16 @@ protected void doClose(final boolean cleanUp) {
     closeKafkaStreams();
 
     if (cleanUp) {
-      kafkaStreams.cleanUp();
+      cleanUpKafkaStreams();
```

Comment:
it seems like stop/start aren't going to do anything except call the close callback - can we just override them to do nothing except for call the close callback?

also note #6068 I refactor this class a bit to make the close/stop a little clearer.

Reply:
Done